tests/internal/remoteconfig/test_remoteconfig.py

```diff
@@ -21,7 +21,6 @@
 from ddtrace.internal.remoteconfig.worker import remoteconfig_poller
 from ddtrace.internal.service import ServiceStatus
 from tests.internal.test_utils_version import _assert_and_get_version_agent_format
-from tests.utils import override_env
```

Comment:
Is this from rebase? Removing this import feels a bit odd. It should either cause syntax or linting errors.

Reply:
This was a fix for a poorly done rebase 😓